style.css

```diff
@@ -0,0 +1,631 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* Mobil display version */
+
+@media screen and (max-width: 768px) {
+  #overlay {
+    position: fixed;
+    opacity: 0;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0, 0, 0, 0.253);
+    pointer-events: none;
+  }
+
+  #overlay.active {
+    pointer-events: all;
+    opacity: 1;
+  }
+
+  nav .mainMenu,
+  .closeMenu {
+    display: none;
+    font-size: 15px;
+    cursor: pointer;
+  }
+
+  nav .top-bar {
+    display: none;
+  }
+
+  nav .closeMenu {
+    display: block;
+    position: absolute;
+    top: 35px;
+    left: 30px;
+  }
+
+  .mainMenu li {
+    margin-top: 1.5rem;
+  }
+
+  .mainMenu li a {
+    text-decoration: none;
+    color: black;
+  }
+
+  .openMenu {
+    display: block;
+    margin: 2rem;
+    cursor: pointer;
+  }
+
+  nav .mainMenu {
+    height: 100vh;
+    position: fixed;
+    top: 0;
+    right: 50px;
+    left: 0;
+    z-index: 10;
+    flex-direction: column;
+    justify-content: center;
+    transition: top 0.8s ease;
+    display: none;
+    background-color: white;
+    list-style: none;
+    padding-bottom: 6rem;
+  }
+
+  .nav-4 {
+    color: #c7724b !important;
+    border: 3px solid #c7724b;
+    padding: 0.6rem;
+  }
+
+  header {
+    background-image: url("./img/background-h1.jpg");
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+  }
+
+  h1 {
+    font-family: cocogoose, sans-serif;
+  }
+
+  h1,
+  .box-text,
+  .box-event,
+  .date-event,
+  .last-text {
+    margin: 2rem;
+  }
+
+  .box-text {
+    font-family: lato, sans-serif;
+    background-color: rgba(255, 255, 255, 0.644);
+    border: 1px solid #6c3611;
+    box-shadow: 15px 5px 5px #6c3611;
+    padding: 0.2rem;
+    border-radius: 3px;
+  }
+
+  .date-event {
+    font-family: lato, sans-serif;
+    font-weight: bold;
+    font-size: 20px;
+  }
+
+  .last-text {
+    font-family: lato, sans-serif;
+  }
+
+  main {
+    background-image: url("./img/background-food.jpg");
+  }
+
+  h2 {
+    font-family: cocogoose, sans-serif;
+    text-decoration: underline;
+    text-decoration-color: #c7724b;
+    text-decoration-thickness: 10%;
+    text-underline-offset: 0.6rem;
+  }
+
+  main h2 {
+    color: white;
+    text-shadow: white 0.1em 0.1em 0.2em;
+    text-align: center;
+    margin-bottom: 2rem;
+    padding-top: 1.5rem;
+  }
+
+  .box {
```

Comment:
- [ ] Let's improve the contrast of this section. As it is right now is difficult to read. I recommend you increase the opacity of the white images in the background and maybe of the boxes as well.

![image](https://user-images.githubusercontent.com/79114233/142571436-00c0a3d5-58c3-422d-8cd8-a1a63780907e.png)